developer_setup.md

```diff
@@ -299,3 +299,32 @@ If this happens, you may be missing developer tools in your OS X. Try to install
 
 If this happens it may be a Bundler issue. Try running `bundle config specific_platform true`
 and re-try.
+
+* Can't install any gems during bundle install
```

Comment:
Sorry, I no longer have access to the macs :(.

And trying locally, it seems it's trying to just run sudo for me, which is definitely not what I saw on the mac.

(Also, is it really that broad? `bin/setup` takes care of installing bundler, and we don't need to compile native extensions for most of the gems, so.. I can't actually come up with any other reason for *all* gems to fail installing other than permissions.)